backend/src/test/java/com/cruru/util/fixture/ApplyFormFixture.java

```diff
@@ -11,8 +11,8 @@ public static ApplyForm createBackendApplyForm(Dashboard backendDashboard) {
                 "크루루 백엔드 모집 공고",
                 "# 모집공고 설명1 ## 이렇게 지원하세요",
                 "www.cruru.kr/example/recruit/apply/1",
-                LocalDateTime.MIN,
-                LocalDateTime.MAX,
+                LocalDateTime.of(2024, 7, 20, 12, 0, 0),
+                LocalDateTime.of(2024, 7, 27, 10, 0, 0),
```

Comment:
나중에 기간을 검증하는 로직을 추가하게 되었을 때 테스트가 깨질 우려가 있는 것 같습니다.
가까운 시일 말고 닿을 수는 없지만 그리 멀지도 않은 과거와 미래로 기간을 설정하면 어떨까요?

```suggestion
                LocalDateTime.of(2099, 12, 31, 23, 59, 59),
```

Reply:
추후에 `모집 시작 시간은 과거일 수 없습니다` 혹은 `모집 종료 시간이 모집 시작 시간보다 과거일 수 없습니다` 같은 로직이 추가되면 좋겠네요.

Reply:
> 추후에 모집 시작 시간은 과거일 수 없습니다 혹은 모집 종료 시간이 모집 시작 시간보다 과거일 수 없습니다 같은 로직이 추가되면 좋겠네요.

검증 로직 추가했습니다.